Delphi.WebMock.RequestStub.pas

```diff
@@ -46,9 +46,18 @@   TWebMockRequestStub = class(TObject)
       : TWebMockResponse;
     function WithBody(const AContent: string): TWebMockRequestStub; overload;
     function WithBody(const APattern: TRegEx): TWebMockRequestStub; overload;
+    /// <summary>
+    ///   Works with TMultipartFormData Fields
+    /// </summary>
+    /// <example>
+    ///   <code lang="Delphi">// FogBugz submission
+    /// WebMock.StubRequest('POST', '/api')
+    ///     .WithBody('cmd', 'new')
+    ///     .ToRespond.WithBody('&lt;?xml version="1.0" encoding="UTF-8"?&gt;&lt;response&gt;&lt;case ixBug="459" operations="edit,spam,assign,resolve"&gt;&lt;/case&gt;&lt;/response&gt;');</code>
+    /// </example>
```

Comment:
Nice start on the XML Docs. Another thing on my to-do list ;)

Reply:
Like decorating, it never ends :)